DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -2,23 +2,27 @@ import Foundation
 import UIKit
 import AVFoundation
 
+private let dcAccounts = DcAccounts()
 private var encryptedDatabases: [Int: Bool] = [:]
 
+public func getDcAccounts() -> DcAccounts {
+    return dcAccounts
+}
```

Comment:
What's the idea behind this?

Reply:
dcAccounts is a global handle; it is there exactly one time on the app. however, one cannot access such globals by variables across modules, therefore a simple getter

Reply:
What do you think about adding a `shared`-property to the `DcAccounts`-class to make it more object-oriented/iOS-y?

Reply:
would be fine for me as well. i am not that deep in these iOS-y things, so, best is if you push sth. directly to this pr

Reply:
Nah, this is your PR, if I contributed, reviewing my own code wouldn't make a lot of sense to me. I'd refactor this on the go somewhen.